src/styles/main.scss

```
@@ -1,3 +1,51 @@
+@import './utils/variables.scss';
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  width: $width-max-calendar;
+  flex-wrap: wrap;
+  gap: $gap-between-days;
+  padding: $padding-between-days;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    height: $height-day-box;
+    width: $width-day-box;
+    border: 1px solid black;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transition: $transition-during;
```

Move to the `__day`, check, if you unhover - it would be without duration

---

don't use it like that, because it's the same as transition: all 0.5s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```